metric/bigtable/src/main/java/com/spotify/heroic/analytics/bigtable/BigtableAnalyticsModule.java

```diff
@@ -87,13 +91,14 @@ public AnalyticsReporter reporter(final HeroicReporter reporter) {
     public Managed<BigtableConnection> connection(
         final AsyncFramework async, final ExecutorService executorService
     ) {
+        Optional<Integer> batchSize = empty();
```

Comment:
The issue is that we would like the default to be what Google has set in the client library, since one would assume they know better and they would change defaults for a good reason.

We can do this by either initializing the vaule to BulkOptions.BIGTABLE_BULK_MAX_ROW_KEY_COUNT_DEFAULT or not initializing it at all so that the default gets applied instead. I went for the former because I fear the latter will create a dependency between both modules and potentially make the code harder to read/maintain in the long term.

What do you suggest?